### PR TITLE
RFC-7: Updates to the values of timeouts and benchmarks

### PR DESCRIPTION
**Pull request by pdyraga**
Refs https://github.com/keep-network/keep-core/issues/3512

No changes to the mechanism. Updated some parameter values based on the current state of the code. Some changes to the client were applied when this PR was in progress.

### CODE REVIEW

File: docs/rfc/rfc-7.adoc [hunk 1]
```diff
@@ -221,23 +221,23 @@ corrupted data:
 
 - With 1 malicious member in a signing group, we need 2 attempts of the protocol
 in the worst case (`P = (99 choose 51) / (100 choose 51) = 0.49`).
-In the worst case, it takes approximately 2h 30min to sign 15 inputs.
+In the worst case, it takes approximately 3h 30min to sign 15 inputs.
```
Comment (lukasz-zimnoch) [inline docs/rfc/rfc-7.adoc:224]: What's the exact math behind it (and the values below)?

Comment (pdyraga) [reply docs/rfc/rfc-7.adoc:224]: There is a ~7 minutes timeout for a single signing attempt. Each input requires a separate signature.
We have 15 inputs and we need 2 attempts for each input.

`15 * 2 * 7min = 21min = 3h 30min`

File: docs/rfc/rfc-7.adoc [hunk 2]
```diff
@@ -200,8 +200,8 @@ client works on a 2-core machine and there are no more than 5 signing group
 members on a single machine, the bottleneck phase should take no more than
 1 minute.
 
-The announcement phase takes always 7 blocks, so ~1.5 minutes assuming 12s block
-time. With 30 blocks timeout for a single attempt, it leaves more than 3 minutes
+The announcement phase takes always 6 blocks, so ~1.5 minutes assuming 12s block
```
[the remark this answers is not in the document]
Comment (pdyraga) [reply docs/rfc/rfc-7.adoc:203]: 84 minutes = 1 minute 24 seconds 

Which is 1.5 min, we are operating in [0.1 - 0.9] decimals here 😉 "one minute and half"

File: docs/rfc/rfc-7.adoc (hunk 1 above)
Comment (lukasz-zimnoch) [reply docs/rfc/rfc-7.adoc:224]: Ah ok, I used a different timeout hence obtained different results. Makes sense!

File: docs/rfc/rfc-7.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (lukasz-zimnoch) [reply docs/rfc/rfc-7.adoc:203]: What I mean is: 6 blocks assuming 12s per block is 72 seconds which gives 1 minute and 12 seconds which is 1.2 min 😀

[the remark this answers is not in the document]
Comment (pdyraga) [reply docs/rfc/rfc-7.adoc:203]: Ah, right. The announcement phase is 1+5 not 1+6. I'll fix it.

[the remark this answers is not in the document]
Comment (pdyraga) [reply docs/rfc/rfc-7.adoc]: 9e234478f5d57804d9abc348f352c022ed79db80

**Pull request merged**